sql_metadb/derived_tables/po_notes.sql

```diff
@@ -0,0 +1,33 @@
+--metadb:table po_notes
+--Creates a derived table to show all notes attached to purchase orders, including general notes, renewal notes, and notes when purchase order is closed. 
+
+DROP TABLE IF EXISTS po_notes;
+CREATE TABLE po_notes AS
+SELECT
+po.id AS po_id,
```

Comment:
If you would, please consider using indentation to make the query easier to read.  For examples, see `feesfines_accounts_actions.sql` above or the query within: https://github.com/metadb-project/metadb-examples/blob/main/folio/reports/missing_items.sql